src/chrome/__mocks__/chrome.js

```diff
@@ -151,6 +152,27 @@ export const mockChrome = (opts = {}) => {
     },
   };
 
+  const windows = {
+    getAll: (callback) => {
+      // This should really be async, but it complicates the tests horribly
+      callback(openWindows);
+    },
+
+    create: (tabSpec, callback) => {
+      const tab = Object.assign({}, tabSpec, {
+        id: tabCounter++,
+      });
+      openTabs.push(tab);
+      const window = { tabs: [tab] };
```

Comment:
`window` is a keyword in the browser, though I doubt that would be an issue here since it's compiled. Might want to use something else anyway just to be clear.